WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -294,12 +316,20 @@ class NoticePresenter: NSObject {
             }
 
             noticeContainer.noticeView.alpha = WPAlphaFull
-            noticeContainer.bottomConstraint?.constant = -self.window.untouchableViewController.offsetOnscreen
+            noticeContainer.bottomConstraint?.constant = self.onscreenNoticeContainerBottomConstraintConstant()
 
             self.view.layoutIfNeeded()
         }
     }
 
+    private func onscreenNoticeContainerBottomConstraintConstant() -> CGFloat {
```

Comment:
this method could be a computed property instead but what ever you prefer

Reply:
Good idea, @yaelirub. Done in 230580a1a2ea0348ca65289c44c6e9b9739ac590.